docs/static/monitoring/monitoring-internal.asciidoc

```diff
@@ -2,9 +2,11 @@
 [[monitoring-internal-collection]]
 === Collect {ls} monitoring data using internal collectors
 ++++
-<titleabbrev>Internal collection</titleabbrev>
+<titleabbrev>Internal collection (deprecated)</titleabbrev>
 ++++
 
+deprecated[7.6.0]
```

Comment:
@karenzone the deprecation starts at `7.7`

Reply:
@andsel  Here is my understanding based on our follow-up discussion

* The docs should be updated in 7.6  with the deprecation notice in the docs. 
* Then you PR 11511 to log a deprecation line if the user is using an internal collector goes in 7.7
* Therefore, I should change the version number in line 8 to be 7.7.0.  But I should merge this docs notice for master, 7.x, and *7.6*.

Is this correct?

Reply:
Yes sounds good, I ping here @roaksoax to double check

Reply:
lgtm!